release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -465,6 +465,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-scalability-and-performance_{context}"]
 === Scalability and performance
 
+[id="ocp-release-notes-numa-resources-operator-replicas_{context}"]
+==== Configuring NUMA-aware scheduler replicas and high availability
+
+By default in {product-title} {product-version} the NUMA Resources Operator automatically enables high availability (HA) mode. In this mode, the NUMA Resources Operator creates one scheduler replica for each control-plane node in the cluster to ensure redundancy. This default behavior occurs if the `spec.replicas` field is not specified in the NUMAResourcesScheduler Custom Resource (CR). Alternatively, you can explicitly set a specific number of scheduler replicas to override the default HA behavior or disable the scheduler entirely by setting the `spec.replicas` field to `0`.
+
+This feature is available as a technology preview. For more information, see the 
```

Comment:
Something is missing here. Anyway, I'd indicate in the title of the RN that it's a TP feature and remove this sentence